doc/man/man3/notcurses_capabilities.3.md

```diff
@@ -92,6 +94,8 @@ multimedia support capable of decoding videos.
 **notcurses_canutf8** returns **true** if the configured locale uses
 UTF-8 encoding, and the locale was successfully loaded.
 
+**notcurses_canoctant** returns **true** if the heuristics suggest
+that the terminal can properly render Unicode 17 octants. Likewise,
```

Comment:
a bit heavy on the "Likewise," anaphora here. let's see if i can suggest a change through github....

Reply:
I think I'd just kill both `Likewise,`s.

Reply:
sorry to be picky

Reply:
I updated the text.